permission_handler_platform_interface/pubspec.yaml

```
@@ -3,7 +3,7 @@ description: A common platform interface for the permission_handler plugin.
 homepage: https://github.com/baseflow/flutter-permission-handler/tree/master/permission_handler_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 3.1.3
+version: 3.1.4
```

I think we can update this as a new feature release:
```suggestion
version: 3.2.0
```